### PR TITLE
Add stats for Accounts Background Service

### PR DESCRIPTION
**Pull request by brooksprumo**
#### Problem

There are not top-level stats for the Accounts Background Service. Specifically, how long it takes to run each iteration.

#### Summary of Changes

Add stats.

### CODE REVIEW

**Review by HaoranYi — DISMISSED** [review]
lgtm

File: runtime/src/accounts_background_service/stats.rs [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+    #[test]
+    fn test_stats_record() {
+        let mut stats = Stats::default();
+        let runtime = Duration::from_secs(1);
+        stats.record(runtime);
+        assert_eq!(stats.num_iterations, 1);
+        assert_eq!(stats.cumulative_runtime, runtime);
+        assert_eq!(stats.min_runtime, runtime);
+        assert_eq!(stats.max_runtime, runtime);
```
[the remark this answers is not in the document]
Comment (brooksprumo) [reply runtime/src/accounts_background_service/stats.rs]: Done!

File: runtime/src/accounts_background_service/stats.rs [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+impl Stats {
+    /// Record stats from this iteration
+    fn record(&mut self, runtime: Duration) {
+        self.num_iterations += 1;
+        self.cumulative_runtime += runtime;
+
+        if runtime < self.min_runtime {
+            self.min_runtime = runtime;
+        }
+        if runtime > self.max_runtime {
```
[the remark this answers is not in the document]
Comment (brooksprumo) [reply runtime/src/accounts_background_service/stats.rs]: Done!

File: runtime/src/accounts_background_service/stats.rs [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    max_runtime: Duration,
+}
+
+impl Stats {
+    /// Record stats from this iteration
+    fn record(&mut self, runtime: Duration) {
+        self.num_iterations += 1;
+        self.cumulative_runtime += runtime;
+
+        if runtime < self.min_runtime {
```
[the remark this answers is not in the document]
Comment (brooksprumo) [reply runtime/src/accounts_background_service/stats.rs]: Done!

**Review by apfitzge — APPROVED** [review]
lgtm

**Pull request merged**